resources/views/settings/index.blade.php

```diff
@@ -444,10 +444,12 @@
                 }
             });
 
-            $('.voyager-trash').click(function () {
+            $('.panel-actions .voyager-trash').click(function () {
                 var display = $(this).data('display-name') + '/' + $(this).data('display-key');
 
                 $('#delete_setting_title').text(display);
+
+                $('#delete_form')[0].action = '{{ route('voyager.settings.delete', [ 'id' => '__id' ]) }}';
```

Comment:
Can't we combine these 2 lines and only have to set the action once? Something like:
```
$('#delete_form')[0].action = '{{ route('voyager.settings.delete', [ 'id' => '__id' ]) }}'.replace('__id', $(this).data('id'));
```

If this works, there are other instances of this that should be updated, but that can be a separate PR since it's unrelated to #1731